packages/react-charting/src/components/SankeyChart/SankeyChart.test.tsx

```diff
@@ -153,6 +153,119 @@ describe('Sankey Chart snapShot testing', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  describe('number formatting', () => {
+    it('renders Sankey correctly by formatting large numbers', () => {
+      // ARRANGE
+      const data2 = {
+        chartTitle: 'Sankey Chart',
+        SankeyChartData: {
+          nodes: [
+            { nodeId: 0, name: 'First' },
+            { nodeId: 1, name: 'Second' },
+            { nodeId: 2, name: 'Third' },
+            { nodeId: 3, name: 'Fourth' },
+            { nodeId: 4, name: 'Five' },
+            { nodeId: 5, name: 'Six' },
+            { nodeId: 6, name: 'Seven' },
+          ],
+          links: [
+            { source: 0, target: 1, value: 1234567890 },
+            { source: 0, target: 2, value: 100000000 },
+            { source: 0, target: 5, value: 1234 },
+            { source: 0, target: 6, value: 100 },
+            { source: 1, target: 3, value: 1000000000 },
+            { source: 1, target: 4, value: 234567890 },
+            { source: 2, target: 3, value: 1000 },
+            { source: 2, target: 4, value: 9999000 },
+          ],
+        },
+      };
+      const strings: ISankeyChartStrings = {
+        linkFrom: 'source {0}',
+      };
+      const accessibilityStrings: ISankeyChartAccessibilityProps = {
+        linkAriaLabel: '{2} items moved from {0} to {1}',
+        nodeAriaLabel: 'element {0} with size {1}',
+      };
+      const formatNumber = (num: number | undefined) => {
+        if (num === undefined || num === null) {
+          return '';
+        }
+        return num.toLocaleString('en-US', {
```

Comment:
instead of defining your own formatter, could you consider using the Intl.DateTimeFormatOptions.
Refer this usage: https://github.com/microsoft/fluentui/blob/804e2233c02c78d04167609238c7c731f607e7e3/packages/react-charting/src/components/CommonComponents/CartesianChart.types.ts#L368

Reply:
Do u mean [Intl.NumberFormatOptions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#options)? I dont think node and link values in sankey can be dates.

Reply:
Yes. We can start with Intl.NumberFormatOptions. DateTime can be added later as per need.

Reply:
updated